### ISSUE TITLE
About batchnorm and scale layer

### ISSUE THREAD

**Issue by DongNaeSwellfish** [ginn24/Pelee-TensorRT#8]
I have questions about the layer types.
The original Pelee model contains batchnorm and scale layer, but it seems like deploy_iplugin.prototxt in tensorRT does not have that layers.
I want to train my own dataset using pre-trained weights from coco-pretrained pelee model, but the model that can be downloaded from https://github.com/Robert-JunWang/Pelee have batchnorm and scale layers, which is different from TensorRT models. 
If you trained your model that does not have bn and scale layers, could you share the coco pretrained caffemodel(coco only, not voc) please...?
I'm looking forward to your answer
Thank you so much :)

**Comment by cathy-kim** [comment]
Hi. @DongNaeSwellfish
In the paper, Pelee-tensorRT acceleration used Merged BN with Conv. 
The merged model can be downloaded at this link.
(https://drive.google.com/open?id=1AH46csPPufZl3NYwk6xcHDmnYVwmN05e) 
You can also make the merged model with tools/gen_merged_model.py at the link. (https://github.com/Robert-JunWang/Pelee/tree/master/tools)
